cache.md

```diff
@@ -0,0 +1,217 @@
+## cache
+
+- [Overview](#overview)
+- [Basic Usage](#basic-usage)
+- [Advanced Usage](#advanced-usage)
+- [Cache Dependencies](#cache-dependencies)
+
+### Overview
+
+Semaphore `cache` tool helps optimize CI/CD runtime by reusing files that your
+project depends on but are not part of version control. You should typically
+use caching to:
+
+- Reuse your project's dependencies, so that Semaphore fetches and installs
+them only when the dependency list changes.
+- Propagate a file from one block to the next.
+
+Cache is created on a per project basis and available in every pipeline job.
+All cache keys are scoped per project. Total cache size is 9.6GB.
+
+The `cache` tool uses key-path pairs for managing cached archives. An archive
+can be a single file or a directory.
+
+`cache` supports the following options:
```

Comment:
Skip the overview subtitle. Overview should go right after the title, followed by a table of content.

example: https://docs.semaphoreci.com/article/75-debugging-with-ssh-access